### ISSUE TITLE
"Pick Sandbox" is not in the header anymore

### ISSUE THREAD

**Issue by CompuIves** [CompuIves/codesandbox-client#1719]
We need to put Pick Sandbox back. I think it should look something like this:

![Screenshot 2019-04-04 at 13 16 43](https://user-images.githubusercontent.com/587016/55551786-ec12ea80-56db-11e9-8c91-887ce42d096b.png)

**Comment by rabingaire** [comment]
I would love to work on this :)

**Comment by CompuIves** [comment]
That's great!! The button needs to run something like 

```
              signals.modalOpened({
                modal: 'pickSandbox',
              })
```

if I remember correctly.

**Comment by rabingaire** [comment]
Thank you for the hint and Yeah it pickSandboxModel 👍

**Comment by SaraVieira** [comment]
This can only appear if the person is a curator and you can check that in:

```
store.user.curatorAt
```
If this is not null they are one :)

**Comment by rabingaire** [comment]
Oh !! that's the reason that I was getting `You're not a curator` Thank you :)

**Comment by SaraVieira** [comment]
Closed by #1723
